roles/openldap-replication/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+- name: copy templates - replication post oid
+  template: src={{ item }}
+            dest={{ openldap.config_dir }}/{{ item | basename }}
```

Comment:
Why does the item need to be basenamed? Is the template not named the same as the destination file?